spec/index.bs

```diff
@@ -1762,13 +1776,47 @@ enum GPUAddressMode {
 };
 </script>
 
+<dl dfn-type="enum-value" dfn-for=GPUAddressMode>
+    : <dfn>"clamp-to-edge"</dfn>
+    ::
+        Texture coordinates are clamped between 0.0 and 1.0, inclusive.
+
+    : <dfn>"repeat"</dfn>
+    ::
+        Texture coordinates wrap to the other side of the texture.
+
+    : <dfn>"mirror-repeat"</dfn>
+    ::
+        Texture coordinates wrap to the other side of the texture, but the texture is flipped
+        when the integer part of the coordinate is odd. 
+</dl>
+
+{{GPUFilterMode}} describes the behavior of the sampler if the sample footprint does not exactly 
+match one texel.
+
 <script type=idl>
 enum GPUFilterMode {
     "nearest",
     "linear"
 };
 </script>
 
+<dl dfn-type="enum-value" dfn-for=GPUFilterMode>
+    : <dfn>"nearest"</dfn>
+    ::
+        Return the value of the texel nearest to the texture coordinates.
+
+    : <dfn>"linear"</dfn>
+    ::
+        Select two texels in each dimension and return a linear interpolation between their values. 
```

Comment:
nit: This makes it sound like, in *n* dimensions, 2*n* texels are interpolated, but isn't it 2<sup>*n*</sup>?